### DISCUSSION TITLE
Allow coc to see avr include directories

### DISCUSSION THREAD

**Discussion by K4LCIFER** [neoclide/coc.nvim#3534] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Coc doesn't recognize `#include <avr/io.h>` for example. Leading to it not recognizing any  code used from those header files.

**Describe the solution you'd like**
I would like for coc to be able to see these directories. I believe that they are all located in `/usr/avr/include/avr`

**Comment by oblitum** [comment, marked as answer]
You have to configure your language server flags.

**Comment by K4LCIFER** [comment]
> You have to configure your language server flags.

How would I go about doing that to accomplish what I'm looking for?

**Reply by oblitum** [reply]
See its docs, for clangd for example: https://clangd.llvm.org/guides/system-headers.
